src/stockprice.py

```diff
@@ -37,19 +40,31 @@
 es = Elasticsearch(hosts=[{'host': elasticsearch_host, 'port': elasticsearch_port}],
                    http_auth=(elasticsearch_user, elasticsearch_password))
 
+regex = re
+
 class GetStock:
 
     def get_price(self, url, symbol):
-        import re
+
+        eastern_timezone = timezone(timezone_str)
 
         while True:
 
+            if self.isNotLive(eastern_timezone):
+                #logger.info("Stock market is not live. Current time: %s" % datetime.datetime.now(timezone).strftime("%Y-%m-%d %H:%M"))
```

Comment:
`currentTime = datetime.datetime.now(timezone).strftime("%Y-%m-%d %H:%M")
logger.info(f"Stock market is not live. Current time: {currentTime}")`

Reply:
That's from an outdated commit.